var/spack/repos/builtin/packages/hyphen/package.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Hyphen(AutotoolsPackage):
+    """A library of text hyphenation."""
+
+    homepage = "http://hunspell.github.io"
+    url      = "http://downloads.sourceforge.net/hunspell/hyphen-2.8.8.tar.gz"
```

Comment:
```suggestion
    url      = "https://downloads.sourceforge.net/hunspell/hyphen-2.8.8.tar.gz"
```

Reply:
Too bad they don't have release tags on the source at GitHub URL https://github.com/hunspell/hyphen.